src/content/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-and-dt.mdx

```diff
@@ -15,28 +15,32 @@ Watch this short video (approx. 2:15 minutes) to learn how to:
 
 To use distributed tracing, you need at least XCFramework agent version 7.3.0. We recommend you use the most recent agent.
 
-<Callout variant="tip">
-Infinite Tracing is not yet available for mobile monitoring.
-</Callout>
-
-## How to set up distributed tracing [#setup]
+## How to set up distributed tracing [#set-up-dt]
 
 For mobile agents that support this feature, it’s enabled by default. 
 
 If you prefer to turn off distributed tracing, see [iOS feature flag](/docs/mobile-monitoring/new-relic-mobile-ios/api-guides/ios-agent-configuration-feature-flags/#dt).
 
+## Infinite Tracing [#infinite-tracing]
+
+If your mobile apps have some downstream services that use the advanced type of distributed tracing called Infinite Tracing, you should enable Infinite Tracing for your mobile apps. This ensures that your root span (the initiating mobile app) is included in the rest of the spans tracked by Infinite Tracing.
+
+Infinite Tracing uses a trace observer in the cloud to collect all your trace spans. The trace observer uses [tail-based sampling](https://docs.newrelic.com/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#tail-based) to sort through your traces. This means it waits until all your spans arrive before assembling them into traces. Then, it looks through all your traces to find the important ones. 
```

Comment:
I don't think you should be explaining redundant information about how Infinite Tracing works in multiple places. I think we should just be pointing to the infinite tracing main doc if they want to know more about how it works (general concept of avoiding redundancy). You may have a reason for redundantly documenting this but I wanted to point out that idea.

Reply:
@zuluecho9, OK, I removed this description of Infinite Tracing!